.editorconfig

```diff
@@ -10,10 +10,6 @@ indent_size = 4
 insert_final_newline = true
 trim_trailing_whitespace = true
 
-# JS files
-[*.js]
```

Comment:
We should keep this in sync with https://github.com/TYPO3/coding-standards/blob/main/.editorconfig, where 2 spaces are used for indenting JS files. (Also, we should keep `.js` files in the `.editorconfig` so PhpStorm can use this for autormatting.)